src/figuresClasses.ts

```diff
@@ -1,19 +1,98 @@
-export interface Figure {
+function toTwoNumsAfter(num: number): number {
+  return Math.trunc(num * 100) / 100;
+}
+
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea: Function,
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
+
+  color: Color;
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(color: Color, a: number, b: number, c: number) {
+    const theLongestSide = Math.max(a, b, c);
```

Comment:
usually there is no article in names
```suggestion
    const longestSide = Math.max(a, b, c);
```